frontend/src/metabase-lib/join.ts

```diff
@@ -243,7 +243,10 @@ export type PickerInfo = {
   isModel?: boolean;
 };
 
-export function pickerInfo(query: Query, metadata: Joinable): PickerInfo {
+export function pickerInfo(
+  query: Query,
+  metadata: Joinable,
+): PickerInfo | null {
```

Comment:
returns `null` when the joined table/card isn't available, e.g. due to sandboxing

Reply:
> returns `null` when the joined table/card isn't available, e.g. due to sandboxing

This would make a very helpful JSDoc for this function.

Reply:
dd101ac279c4990ab3eefa19591306c642ac92b7